sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/IncrementalExecution.scala

```diff
@@ -47,11 +47,19 @@ class IncrementalExecution(
       sparkSession.sparkContext,
       sparkSession.sessionState.conf,
       sparkSession.sessionState.experimentalMethods) {
-    override def extraPlanningStrategies: Seq[Strategy] =
-      StatefulAggregationStrategy ::
-      FlatMapGroupsWithStateStrategy ::
-      StreamingRelationStrategy ::
-      StreamingDeduplicationStrategy :: Nil
+
+    //  We shouldn't miss the parent's strategies
+    val parentPlanner = sparkSession.sessionState.planner
```

Comment:
Is there a reason to create this val rather than use `sparkSession.sessionState.planner`.  I'm just not sure if its okay to freeze this (probably it is?).